VariantEncoding.md

```diff
@@ -39,13 +39,31 @@ Another motivation for the representation is that (aside from metadata) each nes
 For example, in a Variant containing an Array of Variant values, the representation of an inner Variant value, when paired with the metadata of the full variant, is itself a valid Variant.
 
 This document describes the Variant Binary Encoding scheme.
-[VariantShredding.md](VariantShredding.md) describes the details of the Variant shredding scheme.
+The [Variant Shredding spec](VariantShredding.md) describes the details of shredding Variant values as typed Parquet columns.
 
-# Variant in Parquet
-A Variant value in Parquet is represented by a group with 2 fields, named `value` and `metadata`.
-Both fields `value` and `metadata` are of type `binary`, and cannot be `null`.
+## Variant in Parquet
 
-# Metadata encoding
+A Variant value in Parquet is represented by a group with 2 fields, named `variant_value` and `metadata`.
```

Comment:
@rdblue I think changing to `variant_value` could be trickier, since Spark has already written out data with `value` + `metadata`.

Instead, I proposed changing it the other direction, renaming the shredding one to `value`. That change is part of the change to add the logical type here: https://github.com/apache/parquet-format/pull/460.

What do you think?

Reply:
Yeah, that sounds correct to me. I'll update it here.